Makefile.core.mk

```diff
@@ -57,7 +57,7 @@ LD_EXTRAFLAGS += -X ${GO_MODULE}/pkg/version.buildStatus=${GIT_STATUS}
 IS_FIPS_COMPLIANT ?= false # set to true for FIPS compliance
 ifeq ($(IS_FIPS_COMPLIANT), true)
 	CGO_ENABLED = 1
-	LD_FLAGS = ${LD_EXTRAFLAGS} -s -w
+	LD_FLAGS = ${LD_EXTRAFLAGS} -tags strictfipsruntime -s -w
```

Comment:
This will case conflicts if this line is ever changed upstream. Is it possible to get this somehow to Makefile.vendor?

Reply:
@FilipB I added a new target in the Makefile.vendor.mk and then I will create a MR in the midstream dockerfile.